src/test/python/jointGenerator/rockInput.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python
+import numpy as np
+import sys
+from Joint import Joint
+import math
+
+# Reads input text file containing data on rock volume and joints
+# Outputs text file that can be read by sparkRocks
+
+# Process command line arguments:
+inputFile = open(str(sys.argv[1]), 'r')
+outputFile = str(sys.argv[2])
+
+# Read input file
+data = [line.rstrip('\n') for line in inputFile]
+
+# Read bounds for bounding box: 
+#     First three numbers are lower left corner coordinates, 
+#     last three numbers are upper right corner coordinates
+boundingBox = np.array([float(coordinate) for coordinate in data[0].split(' ')])
+
+# Global origin for block
+origin = np.array([float(coordinate) for coordinate in data[1].split(' ')])
+
+facesData = np.empty([0, 5], dtype = float)
+jointsData = np.empty([0, 3], dtype = float)
+
+# Read in block face and joint data:
+#     Data for block faces specified as follows: Strike, dip, coordinates for point in face
+#     Data for joints specified as follows: Strike, dip, joint spacing
+
+block = True
+for i in range(2, len(data)):
+    if (data[i] == ''): 
+        block = False
+    elif (block == True):
+        facesData = np.append(facesData, 
+                          [np.array([float(point) for point in data[i].split(' ')])], axis = 0)
+    else:
+        jointsData = np.append(jointsData, 
+                           [np.array([float(point) for point in data[i].split(' ')])], axis = 0)
+
+faces = np.empty(0, dtype = object)
+masterJoints = np.empty(0, dtype = object)
+joints = np.empty(0, dtype = object)
+
+# Generate joints that represent the bounds for the rock volume
+for face in facesData:
+    center = np.array([face[2], face[3], face[4]])
+    faces = np.append(faces, Joint(origin, center, face[0], face[1]))
+
+# Generate "master joint set". These will be copied based on joint spacing
+for joint in jointsData:
+    # localOrigin = np.array([boundingBox[0], boundingBox[1], boundingBox[2]])
```

Comment:
Let's get rid of the commented out line unless it's important to keep for some reason.